README.md

```diff
@@ -148,7 +148,10 @@ The Amazon Chime SDK for JavaScript uses WebRTC, the real-time communication API
 
 ## Installation
 
-Make sure you have Node.js version 12 or higher. Node 14 is recommended and supported.
+Make sure you have the following dependency installed:
+- Node.js version 12 or higher. Node 14 is recommended and supported.
```

Comment:
I recognize this was just copied from above, but why is 14 recommended? also isn't it redundant to say its supported?

Reply:
Update to use Node 16 the current LTS version.